indexing/src/main/scala/org/allenai/common/indexing/BuildCorpusIndex.scala

```diff
@@ -36,7 +38,13 @@ class BuildCorpusIndex(config: Config) extends Logging {
 
   /** Get Index Name and Index Type. */
   val esConfig: Config = config[Config]("elasticSearch")
-  val indexName: String = esConfig[String]("indexName")
+  val indexName: String = {
+    // The index name must be appended with the current date.
+    val name = esConfig[String]("indexName")
+    val dateFormat = new SimpleDateFormat("yyyy-MM-dd")
+    name + "-" + dateFormat.format(Calendar.getInstance().getTime())
```

Comment:
This is just a light-weight disambiguation and we don't expect to reindex twice in the same day.  The date itself is arbitrary, so plus or minus 8 hours shouldn't matter.